### ISSUE TITLE
mc admin trace crashes

### ISSUE THREAD

**Issue by SCDealy** [minio/mc#2846]
## Expected behavior
Not crashing
## Actual behavior
Crashes
## Steps to reproduce the behavior
All I had to do was enter the command line:
mc admin trace <server_name>
## mc version

Version: 2019-07-31T17:31:58Z
Release-tag: DEVELOPMENT.2019-07-31T17-31-58Z
Commit-id: 81eb54d14670729a7309b9fffd24a97a083819a9

I had to custom build it for my platform (ARM32) checked out the most recent development version.

I located the source of the problem in the file: admin-trace.go, function: colorizedNodeName
Likely I am seeing the problem and others aren't because I am building for a 32 bit platform.
The issue is that nHashSum can be a negative number resulting in a negative index into the colors array.
```sh
nHashSum := nodeHash.Sum32()
idx := int(nHashSum) % len(colors)
return console.Colorize(fmt.Sprintf("Node%d", colors[idx]), nodeName)
```
I did the following quick fix which works, however, I am not a go programmer (this is my first time even looking at go code) so I am not sure if this should be done differently.
```sh
idx := uint(nHashSum) % uint(len(colors))
```
## System information
Raspberry PI 3B+ running raspbian linux (stretch)
go version go1.12.7 linux/arm

**Comment by harshavardhana** [comment]
We don't test on 32-bit platforms so any issues like should be expected @SCDealy - Feel free to send a PR we will happily accept.

**Comment by kannappanr** [comment]
Closing this as fixed as #2847 has been merged
